spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -301,6 +301,16 @@
     and_I_cannot_publish_or_schedule_the_step_by_step
   end
 
+  scenario "Multiple check failures are each listed in the inset prompt" do
```

Comment:
I wasn't sure what "check failures" meant. Perhaps it could be renamed to "required actions"